wave/src/main/java/org/waveprotocol/wave/client/wavepanel/view/dom/full/Tags.css

```
@@ -4,6 +4,9 @@
  */
 
 @eval panelHeight org.waveprotocol.wave.client.wavepanel.view.dom.full.TagsViewBuilder.CssConstants.PANEL_HEIGHT_CSS;
+@external .mobile-wave-open;
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@external" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)